pyproject.toml

```diff
@@ -41,7 +41,7 @@ pydantic = "^1.10.18"
 PyDispatcher = "^2.0.6"
 pytz = "^2022.6"
 tzlocal = "^4.2"
-mxcubecore = ">=1.154.0"
+mxcubecore = ">=1.156.0"
```

Comment:
Why in this pull request?

Reply:
Because it contains the fix that @jbflo made for the typehints, otherwise the tests fail

Reply:
OK, I see now it is a separate commit, so it is fine.